crates/protocol/src/batch/raw.rs

```diff
@@ -16,10 +16,10 @@ pub struct RawSpanBatch {
     pub payload: SpanBatchPayload,
 }
 
-impl TryFrom<SpanBatch> for RawSpanBatch {
+impl TryFrom<&SpanBatch> for RawSpanBatch {
```

Comment:
TryFrom references are usually horrbile to work with, recommend adding function to`SpanBatch` as well `try_to_raw(&self` or smth

Reply:
Yea good callout. Will follow up with this!